stutterhold/Source/PhaseVocodeur/PhaseVocodeur.cpp

```diff
@@ -5,11 +5,28 @@
     Created: 9 Dec 2020 9:19:32pm
     Author:  Julian Vanasse
 
+ By `push`ing into this class, the input is distributed among overlapping buffers,
+ this permits a frequency-domain processing and reconstruction.
+ 
+ Sound comes out via -- TODO: (?)
+ 
+ You can specify your own processing by overwriting the (?) method.
+ 
   ==============================================================================
 */
 
 #include "PhaseVocodeur.h"
 
+// @JulianArmandVanasse: Where does the sound come out?
+
+/*
+ ==============================================================================
+    Constructors
+ ==============================================================================
+*/

```

Comment:
I added all these hedgerows so I wouldn't get lost in the code. We might want to discuss the organization, subheadings, etc.